### ISSUE TITLE
cli command to provide a CAR file and generate a diff from two roots

### ISSUE THREAD

**Issue by raulk** [filecoin-project/statediff#11]
To replace the library usage here with an execution of the statediff binary:

https://github.com/filecoin-project/lotus/blob/b1d251e9502dab68eaa93d25e4d4843d0470bc77/conformance/runner_test.go#L277-L284

I'm thinking of something along the lines of:

```
$ # statediff --car /path/to/car root_cid_1 root_cid_2
$ statediff --car /path/to/car bafy2bzacedegvywl7nddo6gqzodjh7vbyfzqzdqwzeli6kz3mms3yz2udylru bafy2bzacedgvyh2n6esbu3c552ae35nvbnlb3dyw5lrcoxeteg22yup55uqpq
```

---

Blocking https://github.com/filecoin-project/lotus/issues/3666.

**Comment by willscott** [comment]
ref: #12 
note, that'll use
`statediff --car /path/to/car --preCid bafy2bzacedegvywl7nddo6gqzodjh7vbyfzqzdqwzeli6kz3mms3yz2udylru --postCid bafy2bzacedgvyh2n6esbu3c552ae35nvbnlb3dyw5lrcoxeteg22yup55uqpq`

**Comment by raulk** [comment]
Closed in #12.
